src/utils/fetchClient.ts

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable max-len */
+import { Todo } from '../types/Todo';
+
+const BASE_URL = 'https://mate.academy/students-api';
+
+function wait(delay: number) {
```

Comment:
You may consider adding a return type to your wait function, e.g., 'function wait(delay: number): Promise<void>'.

Reply:
didn't fix it